### PR TITLE
feat(screencast): auto recording for new pages in chromium

### PR DESCRIPTION
**Pull request by yury-s**
#1158

### CODE REVIEW

File: src/server/chromium/crPage.ts [hunk 1]
```diff
@@ -454,6 +458,14 @@ class FrameSession {
       promises.push(this._evaluateOnNewDocument(source));
     for (const source of this._crPage._page._evaluateOnNewDocumentSources)
       promises.push(this._evaluateOnNewDocument(source));
+    // if (._browserContext._options._screencas) {
```
Comment (aslushnikov) [inline src/server/chromium/crPage.ts:461]: stray comment

File: test/screencast.spec.ts [hunk 2]
```diff
@@ -225,24 +226,26 @@ describe('screencast', suite => {
...
+    if (options.CHROMIUM)
+      await page.setViewportSize({width: 640, height: 480});
     await page.goto(server.PREFIX + '/rotate-z.html');
     await toImpl(page)._delegate.startScreencast({outputFile: videoFile, width: 640, height: 480});
     // TODO: in WebKit figure out why video size is not reported correctly for
     // static pictures.
     if (options.HEADLESS && options.WEBKIT)
       await page.setViewportSize({width: 1270, height: 950});
-    await new Promise(r => setTimeout(r, 300));
+    // 300 is not enough for Chromium headful.
```
Comment (aslushnikov) [inline test/screencast.spec.ts:247]: this is very scary.

**Review by aslushnikov — APPROVED** [review]

File: test/screencast.spec.ts (hunk 2 above)
Comment (yury-s) [reply test/screencast.spec.ts:247]: yes, but this is the reality

File: src/server/chromium/crPage.ts (hunk 1 above)
Comment (yury-s) [reply src/server/chromium/crPage.ts:461]: done

**Pull request merged**